README.md

```diff
@@ -10,93 +10,168 @@ providing QUnit-specific wrappers around the helpers contained in
 
 ## Usage
 
-### Setting the resolver
+- [Acceptance Tests](#acceptance-tests)
+- [Component Unit Tests](#component-unit-tests)
+- [Component Integration Tests](#component-integration-tests)
+- [Other Tests](#other-tests)
+- [Ember Data Tests](#ember-data-tests)
+
+### Acceptance Tests
+
+[Ember Guide](http://guides.emberjs.com/v1.13.0/testing/acceptance/)
 
 ```js
-// if you don't have a custom resolver, do it like this:
-setResolver(Ember.DefaultResolver.create({namespace: App}));
+module('Visiting the homepage', {
+  beforeEach: function() {
+    App = startApp();
+  },
+  afterEach: function() {
+    Ember.run(App, App.destroy);
+  }
+});
 
-// otherwise something like:
-import Resolver from './path/to/resolver';
-import {setResolver} from 'ember-qunit';
-setResolver(Resolver.create());
+test('The signup form works', function(assert) {
+  assert.expect(2);
+  
+  visit('/');
+  
+  fillIn('.form>input', 'foo@bar.com');
+  click('.submit');
+  
+  andThen(function() {
+    assert.equal(currentPath(), 'results');
+    assert.equal($('.results>li').length, 2);
+  });
+});
 ```
 
-### Simple example:
+### Component Unit Tests
+
+[Ember Guide](http://guides.emberjs.com/v1.13.0/testing/testing-components/)
 
 ```js
-// tell ember qunit what you are testing, it will find it from the
-// resolver
-moduleForComponent('x-foo', 'XFooComponent');
+moduleForComponent('x-foo', 'XFooComponent', { 
+  unit: true,
+  needs: ['helper:pluralize-string']
+});
 
 // run a test
 test('it renders', function(assert) {
-  assert.expect(2);
+  assert.expect(3);
 
   // creates the component instance
-  var component = this.subject();
+  var subject = this.subject();
   assert.equal(component.state, 'preRender');
```

Comment:
Can you remove the lines checking `component.state`?  They don't work in Ember 2.0.0 and `.state` or `._state` has always been private property.  The added check for `.text()` is a much better assertion.